openmetadata-ui/src/main/resources/ui/src/pages/service/index.tsx

```diff
@@ -863,11 +881,90 @@ const ServicePage: FunctionComponent = () => {
         </div>
       );
     }
-  };
-
-  const getDataModalTab = () => (
-    <DataModelTable data={dataModel} isLoading={isLoading} />
+  }, [
+    isAirflowAvailable,
+    airflowEndpoint,
+    serviceDetails,
+    deleteIngestionById,
+    deployIngestion,
+    handleEnableDisableIngestion,
+    ingestions,
+    ingestionPaging,
+    servicePermission,
+    serviceName,
+    serviceList,
+    serviceFQN,
+    triggerIngestionById,
+    getAllIngestionWorkflows,
+  ]);
+
+  const dataModalTab = useMemo(
+    () => <DataModelTable data={dataModel} isLoading={isLoading} />,
+    [dataModel, isLoading]
   );
+
+  const testConnectionTab = useMemo(() => {
+    return (
+      <>
+        <Space className="w-full my-4 justify-end">
+          <Tooltip
+            title={
+              servicePermission.EditAll
+                ? t('label.edit-entity', {
+                    entity: t('label.connection'),
+                  })
+                : t('message.no-permission-for-action')
+            }>
+            <Button
+              ghost
+              data-testid="edit-connection-button"
+              disabled={!servicePermission.EditAll}
+              type="primary"
+              onClick={goToEditConnection}>
+              {t('label.edit-entity', {
+                entity: t('label.connection'),
+              })}
+            </Button>
+          </Tooltip>
+          {allowTestConn && isAirflowAvailable && (
+            <Tooltip
+              title={
+                servicePermission.EditAll
+                  ? t('label.test-entity', {
+                      entity: t('label.connection'),
+                    })
+                  : t('message.no-permission-for-action')
+              }>
+              <TestConnection
+                connectionType={serviceDetails?.serviceType ?? ''}
+                formData={connectionDetails as ConfigData}
+                isTestingDisabled={isTestingDisabled}
+                serviceCategory={serviceCategory as ServiceCategory}
+                serviceName={serviceDetails?.name}
+                // validation is not required as we have all the data available and not in edit mode
+                shouldValidateForm={false}
```

Comment:
If we already have data validation will be returning true, We cannot skip validation in any mode

Reply:
We are using this TestConnection on the Service Detail page and we are directly hitting the API here, so don't need to validate here!
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/66266464/235918834-27d5c8b9-a3a0-46ec-a800-f7e9b65a0695.png">